bridge/client/app/_components/ktb-evaluation-details/ktb-evaluation-details-utils.ts

```diff
@@ -102,3 +105,23 @@ export function createDataPoints(evaluationHistory: Trace[]): IDataPoint[] {
     .sort((a, b) => a.yElement.localeCompare(b.yElement));
   return [...scores, ...sortedResults];
 }
+
+export function parseSloOfEvaluations(evaluationTraces: Trace[]): void {
+  const unparsedEvaluations = (trace: Trace): trace is Trace & { data: { evaluation: IEvaluationData } } =>
+    !!trace?.data?.evaluation?.sloFileContent && !trace.data.evaluation.sloFileContentParsed;
+
+  for (const evaluationTrace of evaluationTraces.filter(unparsedEvaluations)) {
+    parseSloFile(evaluationTrace.data.evaluation);
+  }
+}
+
+function parseSloFile(evaluation: IEvaluationData): void {
+  try {
+    evaluation.sloFileContentParsed = parseYaml(atob(evaluation.sloFileContent)) as SloConfig;
```

Comment:
Do we actually need the parsed file content after this function? Could this be replaced with a simple `evaluation.parsed = true`?